packages/cdc-react/src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render, screen } from "@testing-library/react";
+
+import {
+  Table,
+  TableHead,
+  TableRow,
+  TableHeader,
+  TableBody,
+  TableDataCell,
+  Pill,
+} from "../";
+
+import { Icons } from "@us-gov-cdc/cdc-react-icons";
+
+const mockIndexes = [];
+
+for (let index = 0; index < 100; index++) {
+  mockIndexes.push(index);
+}
+
+const data = mockIndexes.map((item: number) => {
+  return {
+    index: item,
+    checked: false,
+    fileName: "Test File Name " + (item + 1),
+    event: "Test Event Name " + (item + 1),
+    uploadStatus: {
+      color: "busy",
+      label: "Uploading",
+    },
+    submitted: {
+      when: "Just Now",
+      timestamp: "3:45pm 11/17/2023",
+    },
+    details: "detailed info",
+  };
+});
+
+describe("Table component", () => {
+  it("should render a table with data", () => {
+    render(
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableHeader className="table-header-checkbox">
```

Comment:
Having class names that are aware of the content within seems weird to me.  I think it would be better of the content itself worried about it's style.

Reply:
Updated